src/coreclr/debug/di/shimprocess.cpp

```diff
@@ -17,10 +17,6 @@
 #include <limits.h>
```

Comment:
@dotnet/dotnet-diag Do we still need the shimprocess bridge for anything? Should the whole shimprocess.* be deleted?

Reply:
Another good question for @noahfalk.